config/services.php

```diff
@@ -34,6 +34,13 @@
     'google' => [
         'client_id' => env('GOOGLE_CLIENT_ID'),
         'client_secret' => env('GOOGLE_CLIENT_SECRET'),
-        'redirect' => env('APP_URL').'/login/google/callback',//env('GOOGLE_REDIRECT'),
-     ],
+        'redirect' => env('APP_URL') . '/login/google/callback', //env('GOOGLE_REDIRECT'),
+    ],
+
+    'facebook' => [
+        'client_id' => env('FACEBOOK_CLIENT_ID'),
```

Comment:
please add the two new environment variable in the .ev.example so on the further deployment it will be noticed.